docs/UserGuide.md

```diff
@@ -415,6 +415,41 @@ Example:
 
 ![UiFindName](images/UG-screenshots/UiFindName.png)
 
+#### Find by Email:
+
+Finds all students with a particular email.
+
+Formats: `find e/EMAIL
+
+- The search is case-insensitive. e.g., `ghost@woods.com` will match `ghOst@woOds.com`.
```

Comment:
this looks like zero 0 instead of capital O letter but I am not sure if this is my laptop problem